doc/classes/RenderingServer.xml

```diff
@@ -3084,16 +3068,16 @@
 		<constant name="ARRAY_TEX_UV2" value="5" enum="ArrayType">
 			Array is an UV coordinates array for the second UV coordinates.
 		</constant>
-		<constant name="ARRAY_BONES" value="6" enum="ArrayType">
+		<constant name="ARRAY_BONES" value="10" enum="ArrayType">
```

Comment:
Missing intermediate bindings?

Reply:
Fixed. You systematically forget to bind enum values in `rasterizer_server.cpp` :)